package.json

```diff
@@ -79,6 +79,7 @@
     "ng2-dragula": "1.2.1",
     "ng2-dropdown": "0.0.15",
     "ng2-modal": "0.0.22",
+    "ng2bln-count-pipe": "^1.1.0",
```

Comment:
@SMahil This lib should not have the ^ on it. That will auto-update the version.